scilpy/tracking/propagator.py

```diff
@@ -122,8 +122,8 @@ def finalize_streamline(self, last_pos, v_in):
         # interpolation of a new direction). Ex of use: if stopped because it
         # exited the (WM) tracking mask, reaching GM a little more.
 
-        # In this abstract class, tracking_information only contains the last
-        # tracking direction.
+        # In this abstract class, tracking information only consists of the
+        # last tracking direction.
```

Comment:
You can actually remove that line. At some point, v_in was called tracking_information.